scipy/stats/_stats_py.py

```diff
@@ -2334,6 +2348,7 @@ def cumfreq(a, numbins=10, defaultreallimits=None, weights=None):
 
     Examples
     --------
+    >>> import numpy as np
     >>> import matplotlib.pyplot as plt
     >>> from numpy.random import default_rng
```

Comment:
Could you actually remove this and bellow do `rng = np.default_rng()`? The backstory is that it would "fix" another problem which is that we rewrite on the fly this line to add a seed.

Reply:
@tupui Just to clarify - if I remove `import numpy as np` here, then wouldn't `rng = np.default_rng()` not work since it references numpy?

Reply:
Sorry for the confusion. Here I am only referring to this line `>>> from numpy.random import default_rng`. What you did is correct, this is an extra thing I am asking 😃

Reply:
Got it! Can do. 🙂

Reply:
Thank you!